_data/releases.csv

```diff
@@ -1,301 +1,300 @@
+v21.1.18,v21.1,2022-04-12,false,False,Production,False,go1.15,53cb1ed8f2f42376ad76d4888d582c88685b2820,cockroachdb/cockroach,false,false,false
+v21.2.9,v21.2,2022-04-13,false,False,Production,False,go1.16,eafda969629a6c883d645ffbcbbcb94a91adcb40,cockroachdb/cockroach,false,false,false
+v22.1.0-beta.3,v22.1,2022-04-18,false,False,Testing,False,go1.17,0509bc6c07ceed6d77d8cd088cb79d4a0e8f38ed,cockroachdb/cockroach-unstable,false,false,false
+v22.1.0-beta.4,v22.1,2022-04-26,false,False,Testing,False,go1.17,b89ce4cb855227c93e1fff6e7367e870fc9f2fef,cockroachdb/cockroach-unstable,false,false,false
+v21.2.10,v21.2,2022-05-02,false,False,Production,False,go1.16,39511c6a4d0bbb580b7ff6f5c0e1e77664474efb,cockroachdb/cockroach,false,false,false
+v22.1.0-beta.5,v22.1,2022-05-03,false,False,Testing,False,go1.17,24ba4d7211aa9fb9cea18f856c693bc8f592c8f6,cockroachdb/cockroach-unstable,false,false,false
+v22.1.0-rc.1,v22.1,2022-05-09,false,False,Testing,False,go1.17,5b78463ed2e7106a8477b63fa837564ad02bb510,cockroachdb/cockroach-unstable,false,false,false
+v21.1.19,v21.1,2022-05-09,false,False,Production,False,go1.15,bf2bef135b5be9c73ccc445f2d996781ee59e94f,cockroachdb/cockroach,false,false,false
+v21.2.11,v21.2,2022-05-23,false,False,Production,False,go1.16,b05bab30da2ff789dc232da4fac771c01a72a87f,cockroachdb/cockroach,false,false,false
+v22.1.0,v22.1,2022-05-24,false,False,Production,False,go1.17,5b78463ed2e7106a8477b63fa837564ad02bb510,cockroachdb/cockroach,true,false,false
+v22.1.1,v22.1,2022-06-06,false,False,Production,False,go1.17,242d13b1191e31a25a36ed9cd7abc5f5b24cb510,cockroachdb/cockroach,true,false,false
+v21.2.12,v21.2,2022-06-06,false,False,Production,False,go1.16,4f97a101e6c1ba12f1ff133d69beee4333b7216e,cockroachdb/cockroach,false,false,false
+v22.1.2,v22.1,2022-06-22,false,False,Production,False,go1.17,fc456a26830067b6dfbb8cd87e093a28d9b833d1,cockroachdb/cockroach,true,false,false
+v21.2.13,v21.2,2022-07-05,false,False,Production,False,go1.16,a3c796b1a49394d161ccc50fe4c05766d4d5b626,cockroachdb/cockroach,false,false,false
+v22.1.3,v22.1,2022-07-11,false,False,Production,False,go1.17,2ca74540cc1603ff810b43399846bb5b7caba651,cockroachdb/cockroach,true,false,false
+v22.1.4,v22.1,2022-07-19,false,False,Production,False,go1.17,3c6c8933f578a7fd140e24a603d6ec64c6b7a834,cockroachdb/cockroach,true,false,false
+v22.1.5,v22.1,2022-07-28,false,False,Production,False,go1.17,a30a663cbd9323d34d50f343dd038af64671e25f,cockroachdb/cockroach,true,false,false
+v21.2.14,v21.2,2022-08-01,false,False,Production,False,go1.16,8f4dc943d39780a07dff855151eb3ac217064bbe,cockroachdb/cockroach,false,false,false
+v22.1.6,v22.1,2022-08-23,false,False,Production,False,go1.17,760a8253ae6478d69da0330133e3efec8e950e4e,cockroachdb/cockroach,false,false,false
+v22.2.0-alpha.1,v22.2,2022-08-30,false,False,Testing,False,go1.17,3688055c09cd32f1186624a1ccc3fe35e5a0cffd,cockroachdb/cockroach-unstable,true,true,true
```

Comment:
Can you set `mac_arm` to `false` for all version - we haven't started publishing the binaries yet.

Reply:
I'd like to leave it on for the latest 22.2 release just so we have something to look at in the preview, but I'll remove it for this others. Can you confirm which version we will start to publish ARM binaries for Mac?

Reply:
If everything goes as expected, beta4 will be the first release with mac silicone.